publications-2018.bib

```diff
@@ -990,3 +990,16 @@ @PhdThesis{SafinThesis
 }
 
 @Comment{jabref-meta: databaseType:bibtex;}
+
+
+@article{
+  author = {Ira Neitzel, Thomas Wick, Winnifried Wollner},
+  title = {An Optimal Control Problem Governed by a Regularized Phase-field Fracture Propagation Model. Part II The Regularization Limit},
```

Comment:
And here you need to write
```suggestion
  title = {An Optimal Control Problem Governed by a Regularized Phase-field Fracture Propagation Model. {P}art {I}{I} {T}he Regularization Limit},
```
or it will happily make the letters not enclosed in braces into lower-case -- I know, that's stupid, but that's just what bibtex does...